src/pipecat/transports/base_input.py

```diff
@@ -138,17 +138,19 @@ async def _handle_interruptions(self, frame: Frame):
     # Audio input
     #
 
-    async def _vad_analyze(self, audio_frames: bytes) -> VADState:
+    async def _vad_analyze(self, audio_frame: InputAudioRawFrame) -> VADState:
         state = VADState.QUIET
         vad_analyzer = self.vad_analyzer()
         if vad_analyzer:
+            logger.trace(f"{self}: analyzing VAD on {audio_frame}")
             state = await self.get_event_loop().run_in_executor(
-                self._executor, vad_analyzer.analyze_audio, audio_frames
+                self._executor, vad_analyzer.analyze_audio, audio_frame.audio
             )
+            logger.trace(f"{self}: done analyzing VAD on {audio_frame}")
```

Comment:
These two might generate a lot of logs but they might be useful to know if something is hanging on the VAD